src/mediaplayer.cs

```diff
@@ -870,7 +871,7 @@ interface MPMoviePlayerController : MPMediaPlayback {
 		bool ShouldAutoplay { get; set; }
 
 		[Export ("useApplicationAudioSession")]
-		[Availability (Introduced = Platform.iOS_3_2, Deprecated = Platform.iOS_6_0)]
+		[Availability (Deprecated = Platform.iOS_9_0, Message = "Use 'AVPlayerViewController' (AVKit) instead.")]
```

Comment:
@VincentDondain which one is the correct one?

`"Use 'AVPlayerViewController' (AVKit) instead."`

or

`"Use 'AVPlayerViewController (AVKit)' instead."`